app/controllers/triannon/annotations_controller.rb

```diff
@@ -68,12 +73,12 @@ def create
         # it's from app html form
         params.require(:annotation).permit(:data)
         if params["annotation"]["data"]
-          @annotation = Annotation.new({:data => params["annotation"]["data"]})
+          @annotation = Annotation.new({:data => params["annotation"]["data"], :root_container => params[:anno_root]})
```

Comment:
I can't tell from this diff if your style is new-style hashes or old-style hashes.. seems like you have a mix of both. Is this a good time to pick one?

Also, the curly braces aren't needed here, if you don't want them.